### ISSUE TITLE
ModuleNotFoundError: No module named 'basis_cuda'

### ISSUE THREAD

**Issue by back2yes** [rusty1s/pytorch_geometric#29]
Hi, thanks for your excellent work! 
I am using Ubunt 16.04, cuda 9.2 and pytorch 0.4.1
When importing torch_geometric by: 
`from torch_geometric.nn import GCNConv`

I met the following error:
`ModuleNotFoundError: No module named 'basis_cuda'`

Currently I can work with my data on CPU, but maybe using GPU may accelerate things up.
Could you give me some advice about this?

**Comment by rusty1s** [comment]
Hi,
it seems like the subpackages were not installed with CUDA support. Please try to reinstall them following the installation instructions and report back.

**Comment by rusty1s** [comment]
Yeah, thanks. I Found the error in torch_sparse in current master. Will fix it ASAP, in the meantime, please try to reinstall from pip:
pip uninstall torch-sparse
pip install —upgrade torch-sparse

**Comment by back2yes** [comment]
@rusty1s thanks for your quick reply, I tried removing everything and reinstall from pip, but with same error
